install.sh

```diff
@@ -55,22 +54,22 @@ instructions() {
 main() {
     banner
 
+    TMPDIR=$(mktemp -d)
+    cd "${TMPDIR}"
+
     header "Downloading binary from Github"
-    URL="https://github.com/pior/dad/releases/download/${VERSION}/dad-$(make_variant)"
-    TMPFILE=`mktemp`
-    curl -L -# --fail "${URL}" -o "${TMPFILE}"
+    BINARY="dad-$(make_variant)"
+    URL="https://github.com/pior/dad/releases/download/${VERSION}/${BINARY}"
+    curl -L -# --fail "${URL}" -o "${BINARY}"
+    curl -L -# --fail "${URL}.sha256" -o "${BINARY}.sha256"
 
-    header "Downloading SHA from Github"
-    expected_hash=$(curl -LsS --fail "${URL}.sha256")
-    downloaded_hash=$(shasum -a 256 "${TMPFILE}")
-    echo "Expected hash   : ${expected_hash}"
-    echo "Downloaded hash : ${downloaded_hash}"
-    read -p "Correct? [enter]"
+    header "Verify SHA256 checksum"
+    shasum -c "${BINARY}.sha256"
 
     header "Installing to ${DEST}"
-    sudo install "${TMPFILE}" "${DEST}/dad"
+    sudo install "${BINARY}" "${DEST}/dad"
 
-    [[ -e "${TMPFILE}" ]] && unlink "${TMPFILE}"
+    rm -rf "${TMPDIR}"
```

Comment:
Should we just leave the tmpdir to avoid doing a `rm -rf` ?

Reply:
I think that's quite common to keep those tempdir and anyway something should clean this up at one point automatically no?

Reply:
i don't like using `-rf` in a script

Reply:
Removed